Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -606,21 +607,24 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
   }
 
   // vector of FT0 amplitudes
-  std::vector<float> vAmplitudes(208, 0.);
+  int nFT0Channels = o2::ft0::Geometry::Nchannels;
+  int nFT0ChannelsAside = o2::ft0::Geometry::NCellsA * 4;
+  std::vector<float> vAmplitudes(nFT0Channels, 0.);
   // filling FT0 table
   for (auto& ft0RecPoint : ft0RecPoints) {
     const auto channelData = ft0RecPoint.getBunchChannelData(ft0ChData);
     // TODO: switch to calibrated amplitude
     for (auto& channel : channelData) {
       vAmplitudes[channel.ChId] = channel.QTCAmpl; // amplitude, mV
     }
-    float aAmplitudesA[96];
-    float aAmplitudesC[112];
-    for (int i = 0; i < 96; i++) {
-      aAmplitudesA[i] = truncateFloatFraction(vAmplitudes[i], mT0Amplitude);
-    }
-    for (int i = 0; i < 112; i++) {
-      aAmplitudesC[i] = truncateFloatFraction(vAmplitudes[i + 96], mT0Amplitude);
+    float aAmplitudesA[nFT0ChannelsAside];
+    float aAmplitudesC[133];
```

Comment:
In this way you are introducing a dummy channels. Is not it better to introduce at once a proper mapping?
I thought something like 
````
int HW2SW(int id) const { return id < 200 ? id : (id>203 ? id - 4 : -1); }
int SW2HW(int id) const { return id<200 ? id : id + 4; }
````
should be enough.
Anyway, if it is more complex, I hope this solution is indeed temporary and will be overridden by a proper mapping.

Reply:
Alla, I would go for the general solution.

Reply:
General solution takes time